gabrielaEnache/src/main/java/oop/store/AbstractOrder.java

```diff
@@ -0,0 +1,106 @@
+package oop.store;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class AbstractOrder implements Order {
+    protected final long id;
+    protected final LocalDateTime data;
+    protected List<Item> items;
+
+    protected AbstractOrder(long id, List<Item> items) {
```

Comment:
by adding the id as a constructor parameter means that you will always call the super class with id parameter within, however it can be generated and managed within the AbstractOrder class and to leave the constructor with only one parameter.

Reply:
Fixed